### ISSUE TITLE
Multiple GPU training

### ISSUE THREAD

**Issue by WilliamLwj** [kamenbliznashki/chexpert#6]
Hi, I am trying to train the model for multiple epochs on two GPUs. Is there a way for me to specify multiple "--cuda" values so that I can use multiple GPUs?

**Comment by kamenbliznashki** [comment]
Hi - the code is only for single GPU training as it is. The easiest way to modify it for multi-GPU is to wrap the model in torch.nn.DataParallel - you can take a look at the pytorch docs [here](https://pytorch.org/docs/stable/generated/torch.nn.DataParallel.html#torch.nn.DataParallel). A faster implementation would be DistributedDataParallel - you can read more about it [here](https://pytorch.org/docs/stable/generated/torch.nn.parallel.DistributedDataParallel.html#torch.nn.parallel.DistributedDataParallel). I used this for training a generative model on multiple GPUs and you can look at that implementation [here](https://github.com/kamenbliznashki/normalizing_flows/blob/master/glow.py). Hope this helps.
